.gitignore

```diff
@@ -1,3 +1,5 @@
+.idea
```

Comment:
this line should be [added to your global `.gitignore` file](http://stackoverflow.com/questions/7335420/global-git-ignore#7335487)